libinstall/CentWeb.sh

```diff
@@ -327,15 +346,19 @@ $INSTALL_DIR/cinstall $cinstall \
     $INSTALL_DIR_CENTREON/www >> "$LOG_FILE" 2>&1
 
 $INSTALL_DIR/cinstall $cinstall_opts \
-    -u "$CENTREON_USER" -g "$CENTREON_GROUP" -d 755 -m 644 \
+    -u "$CENTREON_USER" -g "$CENTREON_GROUP" -d 775 -m 775 \
     -p $TMP_DIR/final/www \
     $TMP_DIR/final/www/* $INSTALL_DIR_CENTREON/www/ >> "$LOG_FILE" 2>&1
 check_result $? "$(gettext "Install CentWeb (web front of centreon)")"
 
+cp -f $TMP_DIR/final/www/.htaccess $INSTALL_DIR_CENTREON/www/ >> "$LOG_FILE" 2>&1
+$CHOWN $WEB_USER:$WEB_GROUP $INSTALL_DIR_CENTREON/www/.htaccess
+$CHMOD 775 $INSTALL_DIR_CENTREON/www/.htaccess
```

Comment:
Wouldn't 644 be sufficient ?

Reply:
nope, I don't think so

Reply:
Could you explain why a configuration file needs +x ?

And why does the group need +w ? Is .htaccess overwritten by any application ?

Reply:
the .htaccess is modified somewhere else later... I can't find where.
But if I set it to 644, I'm stuck on the login screen and I can't access the IHM.